CHANGELOG.rst

```diff
@@ -7,6 +7,18 @@ The format is based on `Keep a Changelog`_,
 and this project adheres to `Semantic Versioning`_.
 
 
+Version 2.2.2 (2025-01-23)
+--------------------------
+
+* Added: support for Python 3.13
```

Comment:
I wonder whether the information that the Artifactory backend is incompatible with 3.13 should also go to other visible locations -  One could think of `install.rst` or `pyproject.toml`.   Or is that handled in the installation process?

Reply:
In `pyproject.toml` it is also handled:

```toml
[project.optional-dependencies]
artifactory = [ 
    'dohq-artifactory >=1.0.0; python_version < "3.13"',
]
minio = [ 
    'minio',
]
all = [ 
    'dohq-artifactory >=1.0.0; python_version < "3.13"',
    'minio',
]
```

Reply:
Yes it's there - sorry. I overread it.  
Then  I will go ahead with approval quickly.